config/functions.cfg

```diff
@@ -161,6 +163,34 @@ function go_lang {
 
 }
 
+function update_go_version_from_config {
+
+  GO_VERSION_CONFIG_PATH=$GOPATH/src/github.com/multiversx/mx-chain-$ENVIRONMENT-config/goVersion
```

Comment:
Don't say GO_VERSION_CONFIG_PATH ... it's misleading... say GO_VERSION_CONFIG_FILE maybe